dockstore-webservice/src/main/java/io/dockstore/webservice/core/Event.java

```diff
@@ -190,7 +194,8 @@ public void setType(EventType type) {
         CREATE_COLLECTION,
         MODIFY_COLLECTION,
         REMOVE_FROM_COLLECTION,
-        ADD_TO_COLLECTION
+        ADD_TO_COLLECTION,
+        ADD_TO_ENTRY
```

Comment:
What is ADD_TO_ENTRY?

Reply:
Tag/Version

Reply:
I think it should be ADD_VERSION_TO_ENTRY as here it is unclear what is being added to entry.

Reply:
I guess ADD_TO_COLLECTION has the same issue, though that is my bad.